dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -0,0 +1,93 @@
+module Pd
+  class WorkshopDailySurveyController < ApplicationController
+    include WorkshopConstants
+    include JotForm::EmbedHelper
+
+    # Require login
+    authorize_resource class: 'Pd::Enrollment'
+
+    # General workshop daily survey.
+    # GET '/pd/workshop_survey/day/:day'
+    # Where day 0 is the pre-workshop survey, and days 1-5 are the 1st through 5th sessions (index 0-4)
+    #
+    # The JotForm survey, on submit, will redirect to the new_faciliator route (see below)
+    # for the relevant session id.
+    # The pre-workshop survey, which has no session id, will redirect to thanks.
+    def new
+      workshop = Workshop.where(subject: SUBJECT_SUMMER_WORKSHOP).enrolled_in_by(current_user).nearest
```

Comment:
I did a double take here because I thought this was the old workshop model. Not a problem but tech debt for us to do

Reply:
Ha, oh yeah. I want to clean up many of our namespaced classes to be contained in modules so we don't need to repeat the prefix everywhere. But for now, since we also have an old `Workshop` class this could be confusing. I'm happy to add the prefix for now if you think it'll be more readable.